packages/storybook/stories/pattern-prefill/va-prefill-signed-in.stories.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {
+  VaAlert,
+  VaButtonPair,
+} from '@department-of-veterans-affairs/web-components/react-bindings';
+import { getWebComponentDocs, propStructure, StoryDocs } from '../wc-helpers';
+
+const textInputDocs = getWebComponentDocs('va-text-input');
+
+export default {
+  title: 'Patterns/Prefill/Usage Examples/Signed In Prefill Alert',
+  id: 'patterns/va-prefill-signed-in',
+  parameters: {
+    componentSubtitle: 'va prefill pattern signed in',
+  },
+};
+
+const Template = ({
+  slim,
+  status,
+  'disable-analytics': disableAnalytics,
+  visible,
+  'close-btn-aria-label': closeBtnAriaLabel,
+  closeable,
+  'full-width': fullWidth,
+  headline,
+  onCloseEvent,
+  children,
+}) => {
+  return (
+    <>
+    <div style={{ padding: '20px' }}>
+      <div class="schemaform-title">
+        <h1 data-testid="form-title">Request a Board Appeal</h1>
+        <div class="schemaform-subtitle" data-testid="form-subtitle">
+          VA Form 10182 (Notice of Disagreement)
+        </div>
+      </div>
+      <div className="vads-u-padding-y--2">
+        <VaAlert
+          slim={slim}
+          status={status}
+          disableAnalytics={disableAnalytics}
+          visible={visible}
+          closeBtnAriaLabel={closeBtnAriaLabel}
+          closeable={closeable}
+          fullWidth={fullWidth}
+          onCloseEvent={onCloseEvent}
+        >
+          {headline}
+          {children}
+        </VaAlert>
+      </div>
+      <a href="#start" class="vads-c-action-link--green">
```

Comment:
Can `va-link-action` be used instead? 

https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs

Reply:
done